src/main/java/com/spotify/reaper/ReaperApplication.java

```diff
@@ -60,25 +65,43 @@
 public class ReaperApplication extends Application<ReaperApplicationConfiguration> {
 
   static final Logger LOG = LoggerFactory.getLogger(ReaperApplication.class);
-
+  public final static UUID reaperInstanceId = UUID.randomUUID();
+  private static String reaperInstanceAddress;
+  private final ScheduledExecutorService scheduler = Executors.newScheduledThreadPool(1);
+  
   private AppContext context;
 
   public ReaperApplication() {
     super();
     LOG.info("default ReaperApplication constructor called");
     this.context = new AppContext();
+    setInstanceAddress();
   }
 
   @VisibleForTesting
   public ReaperApplication(AppContext context) {
     super();
     LOG.info("ReaperApplication constructor called with custom AppContext");
     this.context = context;
+    setInstanceAddress();
   }
 
   public static void main(String[] args) throws Exception {
     new ReaperApplication().run(args);
   }
+  
+  private void setInstanceAddress() {
+    try{
+      this.reaperInstanceAddress = InetAddress.getLocalHost().getHostAddress();
+    } catch(Exception e) {
+      LOG.warn("Cannot get instance address", e);
+      this.reaperInstanceAddress = "127.0.0.1";
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png 'Severity: CRITICAL') Make the enclosing method "static" or remove this set. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS2696)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Change this instance-reference to a static reference. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS2209)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Make this IP "127.0.0.1" address configurable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1313)